lib/signs/utilities/audio.ex

```diff
@@ -103,21 +102,15 @@ defmodule Signs.Utilities.Audio do
     true
   end
 
-  @spec from_sign(Signs.Realtime.t()) ::
-          {nil | Content.Audio.t() | {Content.Audio.t(), Content.Audio.t()}, Signs.Realtime.t()}
+  @spec from_sign(Signs.Realtime.t()) :: {[Content.Audio.t()], Signs.Realtime.t()}
   def from_sign(sign) do
     multi_source? = Signs.Utilities.SourceConfig.multi_source?(sign.source_config)
-    audio = get_audio(sign.current_content_top, sign.current_content_bottom, multi_source?)
 
-    audio_list =
-      case audio do
-        multi_audios when is_tuple(multi_audios) -> Tuple.to_list(multi_audios)
-        single_audio -> List.wrap(single_audio)
-      end
+    audios = get_audio(sign.current_content_top, sign.current_content_bottom, multi_source?)
```

Comment:
Yes, we could just do that slice, but there are layers further down that also benefit (see `normalize` and `VehiclesToDestination.from_headway_message`). I think the value proposition here is having all of this code use a uniform representation, so that it's more flexible and easier to modify down the line.